disperser/dataapi/docs/docs.go

```diff
@@ -645,12 +660,16 @@ const docTemplate = `{
                 },
                 "total_stake": {
                     "description": "deprecated: use TotalStakePerQuorum instead. Remove when the frontend is updated.",
-                    "type": "integer"
+                    "allOf": [
```

Comment:
I'm not sure what happened here. Perhaps I'm using a different version of swaggo?